paymentsheet/src/main/java/com/stripe/android/link/ui/ErrorText.kt

```diff
@@ -71,28 +85,21 @@ internal fun ErrorText(
     modifier: Modifier = Modifier,
     style: ErrorTextStyle = ErrorTextStyle.Medium
 ) {
-    DefaultLinkTheme {
-        // This is also used in the inline signup form in MPE, so we need
-        // to re-apply the theme here.
-        Row(
-            modifier = modifier.background(
-                color = LinkTheme.colors.errorComponentBackground,
-                shape = style.shape
-            ),
-            verticalAlignment = Alignment.CenterVertically
-        ) {
-            Icon(
-                painter = painterResource(id = R.drawable.stripe_link_error),
-                contentDescription = null,
-                modifier = style.iconModifier,
-                tint = LinkTheme.colors.errorText
-            )
-            HyperlinkedText(
-                text = text,
-                modifier = style.textModifier,
-                color = LinkTheme.colors.errorText,
-                style = style.textStyle
-            )
-        }
+    Row(
```

Comment:
We're not using the `ErrorText` view on inline signup (since we don't want the redesign to apply there for now).